### ISSUE TITLE
Looking for input on multiple selection in celltags 1.0

### ISSUE THREAD

**Issue by markellekelly** [jupyterlab/jupyterlab-celltags#183]
Hi all,
The new version of celltags, currently in progress, will have an updated UI in which clicking on a tag automatically selects the corresponding cells within the notebook (currently, an additional button must be pressed to perform this action). Here is an idea of what this might look like:
![image](https://user-images.githubusercontent.com/29132064/57052934-700ab480-6c3f-11e9-851c-97a7dd16658a.png)
I am hoping to get some insight on what users would expect/prefer to happen if, with a tag already selected ("assert"), they clicked another tag (e.g., bootstrap). The main 3 possibilities I am considering are: 
1. Completely deselects the first tag and only considers the newly clicked tag
2. Selects in the document the union of cells (i.e., any cell tagged with either of the tags.)
3. Selects in the document the intersection of cells (any cell tagged with both tags). 
Thanks for your input!
@ellisonbg

**Comment by bjtho08** [comment]
I would say that for a left-click it should be option 1. However pressing a modifier should probably change this behavior, such that e.g. left-clicking while holding down shift would select the union. I don't know if it is necessary to have option 3 available with a modifier. The best way to handle this, I think, would be to add user settings, such that users who want this behavior can enable it and choose the appropriate modifier(s) if multiple behaviors are enabled.

**Comment by hoangthienan95** [comment]
I agree with @bjtho08 ! left-click = select 1, shift + left-click = select multiple
